install/helm/agones/values.yaml

```diff
@@ -146,10 +146,11 @@ agones:
       name: agones-allocator
       pullPolicy: IfNotPresent
 
-
 gameservers:
   namespaces:
   - default
   minPort: 7000
   maxPort: 8000
 
+helm:
```

Comment:
Question - should this be on by default? Or should it be off?

I would have thought "off" would be better? Or no?

Reply:
Generally this PR lacks of changes to `helm.md`. There is a statement about use `helm test agones --cleanup` as a simple test of your installation.

Reply:
Right, but should we be installing this test pod by default on production deployments by default?

This seems to me like this is something people should opt into, rather than opt out. I didn't realise that the helm test would be installing a test pod on each cluster -- or maybe I'm missing something?

Reply:
+1 for making it false by default, to match what this is changing in install.yaml.

Reply:
Agree, I have updated this PR and subj.